src/app/authentication/models/otpModel.go

```diff
@@ -0,0 +1,13 @@
+package models
+
+import "time"
+
+type OTPValidationRequest struct {
+	Email        string    `gorm:"column:email" json:"email"`
+	OTP          int       `gorm:"column:otp" json:"otp"`
+	CreationTime time.Time `gorm:"column:createdAt" json:"createdAt"`
+}
+
+func (OTPValidationRequest) TableName() string {
+	return "users"
+}
```

Comment:
this will also be in src/models/dbModels.go for user model

Reply:
Resolved